scripts/utils.py

```diff
@@ -283,3 +280,16 @@ def isGitRepo(path='./'):
     return runJob('git rev-parse --git-dir > /dev/null 2>&1', cwd=path,
                   show_command=False, show_output=False)
 
+def version_tuple(version_str):
+    return tuple(map(int, version_str.split('.')))
+
+def checkCMakeVersion(minimumRequired=None):
+    cmakeInstallURL = 'https://github.com/I2PC/xmipp/wiki/Cmake-update-and-install'
+    if minimumRequired:
+        try:
+            result = subprocess.check_output(['cmake', '--version'])
```

Comment:
see runjob() function implemented in utils.py

Reply:
substituted